prometheus.yml

```diff
@@ -0,0 +1,8 @@
+global:
+  scrape_interval: 15s
+  evaluation_interval: 15s
+scrape_configs:
+- job_name: curity
+  metrics_path: /metrics
```

Comment:
Curity's and Prometheus' defaults line up here, so this can be deleted for simplicity's sake